src/tests/conformance/status.rs

```diff
@@ -0,0 +1,95 @@
+use std::{num::ParseIntError, time::Duration};
+
+use tempfile::TempDir;
+use tokio::time::{error::Elapsed, sleep, timeout};
+
+use crate::{
+    protocol::{codecs::binary::Payload, proto::TmStatusChange},
+    setup::node::build_stateful_builder,
+    tools::{
+        config::TestConfig,
+        constants::EXPECTED_RESULT_TIMEOUT,
+        rpc::{get_ledger_info, LedgerInfoResponse, RpcResponse},
+        synth_node::SyntheticNode,
+    },
+};
+
+#[tokio::test]
+#[allow(non_snake_case)]
+async fn C010_TM_STATUS_CHANGE_should_receive_ledger_information_by_status_change() {
```

Comment:
this name is unclear, we need something referring to the node

Reply:
Changed the name of the test.